analyzer/tests/unit/test_checker_handling.py

```diff
@@ -199,7 +208,7 @@ def f(checks, checkers):
 
         # Enable alpha checkers explicitly.
         cfg_handler = ClangSA.construct_config_handler(args)
-        cfg_handler.initialize_checkers(checkers, [('alpha', True)])
+        cfg_handler.initialize_checkers(checkers, [('group:alpha', True)])
```

Comment:
why do we need this change? "alpha" prefix group still should work as there is no profile named "alpha"

Maybe have one testcase with 
cfg_handler.initialize_checkers(checkers, [('group:alpha', True)])
and one with 
cfg_handler.initialize_checkers(checkers, [('alpha', True)])

both should enable the same set of checkers.

Reply:
Will do, and also add some more tests in the next patch to cover some more scenarios.